src/screens/TokenDetail.js

```diff
@@ -0,0 +1,216 @@
+
+    const getShortConfigurationString = () => {
+      const configArr = configurationString.split(':');
+      return `${configArr[0]}:${configArr[1]}...${configArr[3]}`;
+    }
+
+    const renderTokenInfo = () => {
+      return (
+        <div className="token-general-info">
+          <p className="mb-2"><strong>UID: </strong>{this.state.token.uid}</p>
+          <p className="mt-2 mb-2"><strong>Name: </strong>{this.state.token.name}</p>
+          <p className="mt-2 mb-2"><strong>Symbol: </strong>{this.state.token.symbol}</p>
+          <p className="mt-2 mb-2"><strong>Total supply: </strong>{hathorLib.helpers.prettyValue(this.state.totalSupply)} {this.state.token.symbol}</p>
+          <p className="mt-2 mb-0"><strong>Can mint new tokens: </strong>{this.state.canMint ? 'Yes' : 'No'}</p>
+          <p className="mb-2 subtitle">Indicates whether the token owner can create new tokens, increasing the total supply</p>
+          <p className="mt-2 mb-0"><strong>Can melt tokens: </strong>{this.state.canMelt ? 'Yes' : 'No'}</p>
+          <p className="mb-2 subtitle">Indicates whether the token owner can destroy tokens, decreasing the total supply</p>
+          <p className="mt-2 mb-4"><strong>Total number of transactions: </strong>{this.state.transactionsCount}</p>
+        </div>
+      );
+    }
+
+    return (
+      <div className="content-wrapper flex align-items-center">
+        <div className='d-flex flex-row align-items-start justify-content-between token-detail-top'>
+          <div className='d-flex flex-column justify-content-between mt-4'>
+            <div className='token-wrapper d-flex flex-row align-items-center mb-3'>
+              <p className='token-name mb-0'>
+                <strong>{this.state.token.name} ({this.state.token.symbol})</strong>
+              </p>
+            </div>
+            {renderTokenInfo()}
+          </div>
+          <div className='d-flex flex-column align-items-center config-string-wrapper mt-4 ml-3'>
+            <p><strong>Configuration String</strong></p>
+            <span ref="configurationString" className="mb-2">
+              {getShortConfigurationString()}
+              <CopyToClipboard text={configurationString} onCopy={this.copied}>
+                <i className="fa fa-clone pointer ml-1" title="Copy to clipboard"></i>
+              </CopyToClipboard>
+            </span> 
+            <QRCode size={200} value={configurationString} />
+            <a className="mt-2" onClick={(e) => this.downloadQrCode(e)} download={`${this.state.token.name} (${this.state.token.symbol}) - ${configurationString}`} href="true" ref="downloadLink">Download <i className="fa fa-download ml-1" title="Download QRCode"></i></a>
+          </div>
+        </div>
+        <p className="mb-4 text-warning">Only the UID is unique, there might be more than one token with the same name and symbol.</p>
```

Comment:
I thought we were gonna add this using a Bootstrap Alert, like the ones we have to say if a tx is valid or voided. It would have a yellow background and sit at the top of the screen, just like the valid/voided alert.

The way it is now looks a bit weird to me.

Reply:
When I coded I've tried both solutions and kept the one I thought was best. But to be honest both are good to me.

I've uploaded a screenshot with the bootstrap alert on the top.

Which one you think is better?

Reply:
I prefer the new one, but I don't mind if we keep the other.